crates/typos-dict/assets/words.csv

```diff
@@ -29561,6 +29598,8 @@ maraudeurs,marauder
 marbels,marbles
 marbleds,marbles
 marchmallows,marshmallows
+marcos,macros
```

Comment:
Trying to decided what to do about this one because Marcos is a common enough of a name

Reply:
Fair point, I’ll remove it